ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -138,9 +138,10 @@ StackLayout {
 
             Component {
                 id: chatLogViewDelegate
+
                 Rectangle {
                     id: chatBox
-                    height: 60 + chatText.height
+                    height: repeatMessageInfo ? 60 + chatText.height : 5 + chatText.height
```

Comment:
This is highly arbitrary and most likely needs to be revisited with proper style guidelines.